src/app.rb

```diff
@@ -0,0 +1,107 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+
+class App
+  def initialize(books = [], people = [], rentals = [])
+    @books = books
+    @people = people
+    @rentals = rentals
+  end
+
+  def list_all_books
+    @books.each_with_index do |_books, index|
+      puts "#{index + 1}) title: #{_books.title} - author: #{_books.author}"
+    end
+  end
+
+  def list_all_people
+    @people.each do |person|
+      puts "[#{person.class}] ID: #{person.id} Name: #{person.name} - Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    flag = true
+    while flag
+      puts 'Do you want to create a student (1) or a teacher (2)? [Input a number]:'
+      choice = gets.chomp.to_i
+      case choice
+      when 1
+        create_student
+        flag = false
+      when 2
+        create_teacher
+        flag = false
+      else
+        puts 'Invalid input. Please try again.'
+      end
+    end
+  end
+
+  def create_student
+    puts 'Age: '
+    age = gets.chomp.to_i
+    puts 'Name: '
+    name = gets.chomp.capitalize
+    puts 'Has parent permission? (y/n): '
+    parent_permission = gets.chomp.downcase == 'y'
+    student = Student.new(age, name, parent_permission: parent_permission)
+    @people << student unless @people.include?(student)
+    puts 'Student created successfully.'
+  end
```

Comment:
- [ ] Good job so far 👍 After creating a student and displaying the created persons, the name of the student does not show. Kindly fix this issue 👍 
- Your console:
![Annotation 2022-06-17 061026](https://user-images.githubusercontent.com/85113567/174229697-d02bf22a-fa0d-4c3a-b753-5e6173859e88.jpg)